goss-testing/tests/ncn/goss-bgp-established-aruba.yaml

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
What is this test checking that is not already being checked by the existing test goss-switch-bgp-neighbor-aruba-or-mellanox.yaml ? Is there a reason not to use the existing test instead of writing a new one?